test/Microsoft.IdentityModel.TestUtils/TestStub.cs

```diff
@@ -1,8 +1,5 @@
-//-------------------------------------------------------------------------------------------------
```

Comment:
Interesting that it didn't have the same copyright format than the others :-)

Reply:
Yes. lol.